README.md

```diff
@@ -1,189 +1,159 @@
-#Properties  
 
+# Property Basics
 
+## Objectives
 
-###Private instance variables within a method 
-#####(or with "local" scope)
-Every variable we create within an instance method has been an "instance variable" by definition. It has "local" scope, because it is limited to being used within the method in which it has been created. In other words, if you were to try to access that variable in another method, even within the same class, the compiler would not recognize it, and your program would not run.
+1. Recognize how properties can simplify code.
+2. Declare a property correctly.
+3. Access a property from outside the class file.
+4. Access a property from within the class file using `self.` ("self dot").  
 
+## The Usefulness of Properties
 
+When writing code, the real power of utilizing properties reveals itself in being able to write methods that can access values stored in properties without needing to receive those values as arguments. Did you notice in the Apples & Holidays lab that every one the "holiday" methods had to accept the dictionary as an argument? 
```

Comment:
Thinking of objects as metaphors to real life... real life things have properties. A car has a behavior like door opening, and properties like being blue, having 4 wheels. A major way we describe things in real life is through their properties. That's also how we distinguish between my iPhone and your iPhone. Both of our phones have the same behaviors...but the properties are different.